### ISSUE TITLE
"Config" parameter not working

### ISSUE THREAD

**Issue by hassaanmomin** [cypress-io/github-action#910]
I've been trying to add/update the baseUrl using the config parameter but nothing seems to work. If I add baseUrl in config it never overwrite it and takes it directly from config file and if I don't add it, it try to open '/' path.

**Cypress Version:** 10.11.0
**Node Version:** 18.16.0

**Workflow snippet:** 
<img width="751" alt="Screenshot 2023-05-11 at 5 30 02 PM" src="https://github.com/cypress-io/github-action/assets/49152065/16ed60c3-413d-4021-a5b3-0af79a7386c5">

**settings being passed that can be seen in the actual run:**
<img width="1054" alt="Screenshot 2023-05-11 at 5 29 05 PM" src="https://github.com/cypress-io/github-action/assets/49152065/913e2db5-02d8-4141-8dd5-836b1e8817f7">

**cypress.config.js:**
<img width="481" alt="Screenshot 2023-05-11 at 5 31 18 PM" src="https://github.com/cypress-io/github-action/assets/49152065/93b081dc-886a-47ae-952b-a7125084af06">

**spec file with cy.visit():**
<img width="556" alt="Screenshot 2023-05-11 at 5 31 38 PM" src="https://github.com/cypress-io/github-action/assets/49152065/59bb1c33-d89d-4ada-830d-6d36e533cbef">

**Comment by MikeMcC399** [comment]
@hassaanmomin 

I don't see any error in the parts that you have posted.

You may be able to find out better what is happening if you enable debugging e.g.

```
  env:
    DEBUG: '@cypress/github-action, cypress:*'
```

Which version of Yarn are you using Classic (v1) or Modern (v2 and later)?

**Comment by hassaanmomin** [comment]
@MikeMcC399 I'm using classic (v1.22.19) and here is the error when I remove the baseUrl from the config file.
![Screenshot 2023-05-11 at 8 56 24 PM](https://github.com/cypress-io/github-action/assets/49152065/14a1b311-8fc4-4f89-a55c-424b5033a9d9)

On adding a placeholder it just takes the value from config, no overwriting happens. Btw let me try again with debugging and share my findings. Thanks

**Comment by hassaanmomin** [comment]
Everywhere I see baseUrl it has value 'null'
![Screenshot 2023-05-11 at 9 14 11 PM](https://github.com/cypress-io/github-action/assets/49152065/dff505dd-6b8c-470d-9f28-8a3489dc08bc)

Here are complete log
[11_Run E2E tests (Cypress).txt](https://github.com/cypress-io/github-action/files/11454332/11_Run.E2E.tests.Cypress.txt)
s:

**Comment by MikeMcC399** [comment]
@hassaanmomin 

I missed the fact that you are using `command`! It is clear in the logs that the other parameters, including `config` are ignored.

Please note the [README > Custom install command](https://github.com/cypress-io/github-action#custom-test-command) where it says:

"Caution: using the action parameter command causes multiple other parameters to be ignored including: auto-cancel-after-failures, browser, ci-build-id, command-prefix, component, config, config-file, env, group, headed, parallel, project, publish-summary, quiet, record, spec and tag."

**Comment by MikeMcC399** [comment]
If you are using Yarn Classic and you have a `yarn.lock` file then `github-action` will automatically install your dependencies. You don't need an extra manual step for this. It also handles the caching.

**Comment by hassaanmomin** [comment]
@MikeMcC399 it worked and thanks for the other tips, they also worked like a charm 🎉
